src/pybind/mgr/volumes/fs/stats_util.py

```diff
@@ -136,16 +141,21 @@ def get_stats(src_path, dst_path, fsh, human=True):
     }
 
 
-class CloneProgressBar:
+class CloneInfo:
 
-    def __init__(self, vc, volname, dst_svname, src_path, dst_path):
-        self.vc = vc
+    def __init__(self, volname, dst_sv_name, src_path, dst_path):
         self.volname = volname
-        self.dst_svname = dst_svname
+        self.dst_sv_name = dst_sv_name
 
         self.src_path = src_path
         self.dst_path = dst_path
 
+
+class CloneProgressBar:
+
+    def __init__(self, vc):
+        self.vc = vc
+        self.clones = []
```

Comment:
This tracks the on-going clones, yes? I see this being referenced in self.update timer callback. Any reason it's done this way rather than listing the clone index?